app/src/main/java/org/mozilla/focus/tiles/HomeTilesManager.kt

```diff
@@ -19,6 +19,7 @@ import org.mozilla.focus.utils.UrlUtils
 import java.util.UUID
 
 private const val PREF_HOME_TILES = "homeTiles"
+private const val BUNDLED_SITES_BLACKLIST = "blacklist"
```

Comment:
Oh, since you have this here, you can delete the earlier unnecessary references.